content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -81,11 +81,11 @@ export class ContextMenuItem extends React.PureComponent {
   render() {
     const {option} = this.props;
     return (
-      <li role="menuitem" className="context-menu-item">
-        <a onClick={this.onClick} onKeyDown={this.onKeyDown} tabIndex="0" className={option.disabled ? "disabled" : ""}>
+      <li role="menuitem" className="context-menu-item" onClick={this.onClick} onKeyDown={this.onKeyDown} tabIndex="0" >
+        <span className={option.disabled ? "disabled" : ""}>
```

Comment:
Now that I look at this - out of curiosity, what's your reasoning for using `<span>` rather than `<button>` here?

Reply:
the `<button>` creates a second tab stop that makes the functionality kind of irritating, because you have to tab twice to move past each menu item - makes it really easy to pick the wrong one.